packages/domain-picker/src/domain-picker/style.scss

```diff
@@ -209,7 +218,7 @@ $accent-blue: #117ac9;
 }
 
 .domain-picker__suggestion-item-name {
-	flex-grow: 2;
+	flex-grow: 1;
```

Comment:
Curious to know the reason for this change as I am not sure what issue it fixes.

Reply:
Good question. It's out of the scope of this PR but it's a very small change so I did it anyways.

`flex-grow: 2;`

![image](https://user-images.githubusercontent.com/17054134/101781114-b7ce0f00-3af7-11eb-9dca-6f888807f0b5.png)

`flex-grow: 1;`
![image](https://user-images.githubusercontent.com/17054134/101781264-dd5b1880-3af7-11eb-84d8-b7aceddeca0e.png)

Reply:
I agree the change is a much needed improvement. Although I think we should try and avoid including out of scope changes into PRs as it will be hard to track where these have been introduced and why if they've no relationship to the PR. We'll leave this one as is though I think. What do you think?

Reply:
Yeah I agree, it was lazy to slide it in, I'll create a quick issue and PR for it.